conn.go

```diff
@@ -428,6 +428,7 @@ const (
 	NoENOBUFS
 	ListenAllNSID
 	CapAcknowledge
+	ExtendAckReporting
```

Comment:
Let's go with "ExtendedAcknowledge" like we talked about on Slack, but otherwise this all LGTM.